src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.SortCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.DateDistantToRecentComparator;
+import seedu.address.model.person.DateRecentToDistantComparator;
+import seedu.address.model.person.PriorityHighToLowComparator;
+import seedu.address.model.person.PriorityLowToHighComparator;
+
+/**
+ * Parses input arguments and creates a new {@code SortByPriorityCommand} object
```

Comment:
Hi @YINZE66 I think you meant `creates a new {@code SortCommand} object` instead of `creates a new {@code SortByPriorityCommand} object`

Reply:
Yes, thanks for reminding.